frontend/lib/Repository/UserRepository/UserGateway.dart

```diff
@@ -21,23 +21,26 @@ class UserGateway {
         .then((value) => user = User(value.tokens!));
   }
 
-  Future<AuthResponse> authenticateUser(AuthenticateForm form) async {
-    String? access = await secureStorage.read(key: accessKey);
-    String? refresh = await secureStorage.read(key: refreshKey);
+  Future<AuthResponse> getFromMemory() async {
+    final String? access = await secureStorage.read(key: accessKey);
+    final String? refresh = await secureStorage.read(key: refreshKey);
 
     if (access != null && refresh != null) {
-      //print("FROM MEMORY"); // for reviewer
       user =
           User(AuthTokenResponse(accessToken: access, refreshToken: refresh));
       return AuthResponse(success: true, tokens: user.tokenResponse);
```

Comment:
Czy tak nie będzie prościej? :thinking: 
```suggestion
      return AuthResponse(
          success: true, 
          tokens: AuthTokenResponse(accessToken: access, refreshToken: refresh)
       );
```